src/commands/serviceConnector/IServiceConnectorContext.ts

```diff
@@ -0,0 +1,10 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { ICreateLinkerContext } from "@microsoft/vscode-azext-serviceconnector";
+import { IContainerAppContext } from "../createContainerApp/IContainerAppContext";
+
+export interface IServiceConnectorContext extends ICreateLinkerContext, IContainerAppContext {
```

Comment:
Nit: A possible cleaner looking alternative (if you don't end up adding any properties):

`export type IServiceConnectorContext = ICreateLinkerContext & IContainerAppContext;`

On a side note, I'm more and more agreeing with Alex on not liking the `I` prefix for context interfaces/types (that's not a change request, more of just a general musing 🙂).